requirements/text.txt

```diff
@@ -1 +1,3 @@
-jiwer==2.2.0
+jiwer>=2.2.0
```

Comment:
any reason the requirement for `jiwer` was changed?

Reply:
@Borda had made this change.

Reply:
it was me... as it was discussed in parallel PR
do you know why we froze this version? I this it is fine for the testing bus as for dependency?

Reply:
I am not sure why we froze it, but if tests passes then I am fine with it :]